_posts/2017-08-18-flipkart.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Flipkart
+sourceURL: http://tech.economictimes.indiatimes.com/news/internet/for-flipkart-this-app-makes-rural-connect/59676200
+image: /images/flipkart.png
+tags:
+  - App Shell
+  - E-commerce
+  - Home Screen
+  - Notifications
+  - Offline
+---
+
+**Flipkart's** PWA is driving **50%** of its new customer acquisition. Nearly **60%** of visitors to the PWA had previously uninstalled the native app primarily to save space.
```

Comment:
Nit: Using straight apostrophe (`'`) while in other post you use a fancy one (`’`). Fancy one is better.